### PR TITLE
Add data point type option to `knx.telegram` trigger

### PR DESCRIPTION
**Pull request by farmio**
## Proposed change

Add data point type option to `knx.telegram` trigger

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/117463
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -164,7 +164,7 @@ The `knx.telegram` trigger can be used to trigger automations on incoming or out
 
 <div class='note'>
 
-This trigger is also provided as a device trigger by the `KNX Interface` device. It supports setting the options in the automation builder UI.
+This trigger is also provided as a device trigger by the `KNX Interface` device. It supports setting the options in the automation builder UI, but doesn't support setting a specific DPT (`type`) to decode the payload as it always relies on project data.
```
Comment (c0ffeeca7) [inline source/_integrations/knx.markdown]:
```suggestion
This trigger is also provided as a device trigger by the `KNX Interface` device. It supports setting the options in the automation builder UI, but doesn't support setting a specific <abbr title="data point type">DPT</abbr> (`type`) to decode the payload as it always relies on project data.
```

File: source/_integrations/knx.markdown [hunk 2]
```diff
@@ -173,6 +173,10 @@ destination:
   description: A group address or a list of group addresses the trigger should listen on. If not set, or an empty list, the trigger will listen on all group addresses.
   type: [string, list]
   required: false
+type:
+  description: If set, the payload will be decoded as given DPT in the trigger data. When not set, the DPT is sorced from project data. KNX sensor types are valid values [KNX Sensor](#sensor) (e.g., "2byte_float" or "percent").
```
Comment (c0ffeeca7) [inline source/_integrations/knx.markdown]:
```suggestion
  description: If set, the payload will be decoded as given DPT in the trigger data. When not set, the DPT is sourced from project data. KNX sensor types are valid values [KNX Sensor](#sensor) (e.g., "2byte_float" or "percent").
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @farmio :+1:

**Review by marvin-w — APPROVED** [review]
Nice, looks good! Thanks @farmio !

Will merge this now as parent is merged.

**Pull request merged**